### ISSUE TITLE
Account Suspended after version update

### ISSUE THREAD

**Issue by mxsxs2** [chatwoot/chatwoot#5505]
**Describe the bug**

I updated our self hosted Chatwoot instance  deployed in Kubernetes with helm.  I followed the instructions to upgrade the helm chart version from 0.9.0 to latest. Now when I try to login as a user I get the "Account Suspended" message in the browser. However, this seems to be misleading as in the logs of the pod I have a 500 error.

I can log into superadmin but not to the normal user area.

**To Reproduce**

1. Have an older version of chatwoot deployed in Kubernetes
2. Follow the upgrade process at [the documentation](https://www.chatwoot.com/docs/self-hosted/deployment/helm-charts)

**Expected behavior**

Users can log in

**Screenshots**

![image](https://user-images.githubusercontent.com/13612429/192537576-4b67e66e-8be7-4980-8a3b-4b420fab103b.png)

**Browser logs**

```
xhr.js:184          GET https://mydomain.com/api/v1/accounts/1/ 500
(anonymous) @ xhr.js:184
e.exports @ xhr.js:20
e.exports @ dispatchRequest.js:39
u.request @ Axios.js:111
a.forEach.u.<computed> @ Axios.js:132
(anonymous) @ bind.js:11
value @ ApiClient.js:38
(anonymous) @ accounts.js:35
u @ videojs.record.js:2868
(anonymous) @ videojs.record.js:3088
L.forEach.e.<computed> @ videojs.record.js:2921
b @ application-1acdc7d569a592978d11.js:2
o @ application-1acdc7d569a592978d11.js:2
(anonymous) @ application-1acdc7d569a592978d11.js:2
(anonymous) @ application-1acdc7d569a592978d11.js:2
(anonymous) @ accounts.js:32
(anonymous) @ vuex.js:778
h.dispatch @ vuex.js:497
dispatch @ vuex.js:416
(anonymous) @ App.vue:89
u @ videojs.record.js:2868
(anonymous) @ videojs.record.js:3088
L.forEach.e.<computed> @ videojs.record.js:2921
x @ application-1acdc7d569a592978d11.js:2
o @ application-1acdc7d569a592978d11.js:2
(anonymous) @ application-1acdc7d569a592978d11.js:2
(anonymous) @ application-1acdc7d569a592978d11.js:2
initializeAccount @ application-1acdc7d569a592978d11.js:2
currentAccountId @ App.vue:77
pn.run @ vue.common.prod.js:1488
En @ vue.common.prod.js:1402
(anonymous) @ vue.common.prod.js:701
Ze @ vue.common.prod.js:673
Promise.then (async)
Xe @ vue.common.prod.js:680
nt @ vue.common.prod.js:705
(anonymous) @ vue.common.prod.js:1478
pn.update @ vue.common.prod.js:1466
de.notify @ vue.common.prod.js:284
set @ vue.common.prod.js:396
(anonymous) @ vue-router.esm.js:2878
(anonymous) @ vue-router.esm.js:2877
we.updateRoute @ vue-router.esm.js:2321
(anonymous) @ vue-router.esm.js:2186
(anonymous) @ vue-router.esm.js:2308
a @ vue-router.esm.js:1947
a @ vue-router.esm.js:1954
Oe @ vue-router.esm.js:1959
(anonymous) @ vue-router.esm.js:2302
a @ vue-router.esm.js:1947
(anonymous) @ vue-router.esm.js:1951
(anonymous) @ vue-router.esm.js:2289
(anonymous) @ vue-router.esm.js:2053
(anonymous) @ vue-router.esm.js:2131
Promise.then (async)
(anonymous) @ vue-router.esm.js:2075
(anonymous) @ vue-router.esm.js:2097
(anonymous) @ vue-router.esm.js:2096
ye @ vue-router.esm.js:2095
(anonymous) @ vue-router.esm.js:2033
A @ vue-router.esm.js:2270
a @ vue-router.esm.js:1950
a @ vue-router.esm.js:1954
(anonymous) @ vue-router.esm.js:1951
(anonymous) @ vue-router.esm.js:2289
GD.window.chatwootConfig.signupEnabled.e.meta.e.meta.requireSignupEnabled.UD.includes.getters @ index.js:87
GD @ index.js:114
(anonymous) @ index.js:135
Promise.then (async)
(anonymous) @ index.js:126
A @ vue-router.esm.js:2270
a @ vue-router.esm.js:1950
Oe @ vue-router.esm.js:1959
we.confirmTransition @ vue-router.esm.js:2297
we.transitionTo @ vue-router.esm.js:2185
Ye.init @ vue-router.esm.js:2873
beforeCreate @ vue-router.esm.js:1314
We @ vue.common.prod.js:638
en @ vue.common.prod.js:1375
e._init @ vue.common.prod.js:1756
Ln @ vue.common.prod.js:1652
window.onload @ application.js:97
load (async)
(anonymous) @ application.js:96
(anonymous) @ application-1acdc7d569a592978d11.js:2
i @ bootstrap:72
(anonymous) @ bootstrap:248
(anonymous) @ application-1acdc7d569a592978d11.js:2
Show 3 more frames
App.vue:91 Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'locale')
    at App.vue:91:1
    at u (videojs.record.js:2868:27)
    at Generator._invoke (videojs.record.js:3088:32)
    at L.forEach.e.<computed> [as next] (videojs.record.js:2921:31)
    at x (application-1acdc7d569a592978d11.js:2:9794297)
    at o (application-1acdc7d569a592978d11.js:2:9796508)
(anonymous) @ App.vue:91
u @ videojs.record.js:2868
(anonymous) @ videojs.record.js:3088
L.forEach.e.<computed> @ videojs.record.js:2921
x @ application-1acdc7d569a592978d11.js:2
o @ application-1acdc7d569a592978d11.js:2
Promise.then (async)
x @ application-1acdc7d569a592978d11.js:2
o @ application-1acdc7d569a592978d11.js:2
(anonymous) @ application-1acdc7d569a592978d11.js:2
(anonymous) @ application-1acdc7d569a592978d11.js:2
initializeAccount @ application-1acdc7d569a592978d11.js:2
currentAccountId @ App.vue:77
pn.run @ vue.common.prod.js:1488
En @ vue.common.prod.js:1402
(anonymous) @ vue.common.prod.js:701
Ze @ vue.common.prod.js:673
Promise.then (async)
Xe @ vue.common.prod.js:680
nt @ vue.common.prod.js:705
(anonymous) @ vue.common.prod.js:1478
pn.update @ vue.common.prod.js:1466
de.notify @ vue.common.prod.js:284
set @ vue.common.prod.js:396
(anonymous) @ vue-router.esm.js:2878
(anonymous) @ vue-router.esm.js:2877
we.updateRoute @ vue-router.esm.js:2321
(anonymous) @ vue-router.esm.js:2186
(anonymous) @ vue-router.esm.js:2308
a @ vue-router.esm.js:1947
a @ vue-router.esm.js:1954
Oe @ vue-router.esm.js:1959
(anonymous) @ vue-router.esm.js:2302
a @ vue-router.esm.js:1947
(anonymous) @ vue-router.esm.js:1951
(anonymous) @ vue-router.esm.js:2289
(anonymous) @ vue-router.esm.js:2053
(anonymous) @ vue-router.esm.js:2131
Promise.then (async)
(anonymous) @ vue-router.esm.js:2075
(anonymous) @ vue-router.esm.js:2097
(anonymous) @ vue-router.esm.js:2096
ye @ vue-router.esm.js:2095
(anonymous) @ vue-router.esm.js:2033
A @ vue-router.esm.js:2270
a @ vue-router.esm.js:1950
a @ vue-router.esm.js:1954
(anonymous) @ vue-router.esm.js:1951
(anonymous) @ vue-router.esm.js:2289
GD.window.chatwootConfig.signupEnabled.e.meta.e.meta.requireSignupEnabled.UD.includes.getters @ index.js:87
GD @ index.js:114
(anonymous) @ index.js:135
Promise.then (async)
(anonymous) @ index.js:126
A @ vue-router.esm.js:2270
a @ vue-router.esm.js:1950
Oe @ vue-router.esm.js:1959
we.confirmTransition @ vue-router.esm.js:2297
we.transitionTo @ vue-router.esm.js:2185
Ye.init @ vue-router.esm.js:2873
beforeCreate @ vue-router.esm.js:1314
We @ vue.common.prod.js:638
en @ vue.common.prod.js:1375
e._init @ vue.common.prod.js:1756
Ln @ vue.common.prod.js:1652
window.onload @ application.js:97
load (async)
(anonymous) @ application.js:96
(anonymous) @ application-1acdc7d569a592978d11.js:2
i @ bootstrap:72
(anonymous) @ bootstrap:248
(anonymous) @ application-1acdc7d569a592978d11.js:2

```

**Server logs**

```
I, [2022-09-27T13:11:11.626768 #1]  INFO -- : source=rack-timeout id=9b948720-c410-40e6-86ae-a600ada774da timeout=15000ms state=ready
I, [2022-09-27T13:11:11.628000 #1]  INFO -- : [9b948720-c410-40e6-86ae-a600ada774da] Started GET "/" for xxx.xxx.xxx.xxx at 2022-09-27 13:11:11 +0000
I, [2022-09-27T13:11:11.631169 #1]  INFO -- : [9b948720-c410-40e6-86ae-a600ada774da] Processing by DashboardController#index as HTML
I, [2022-09-27T13:11:11.638372 #1]  INFO -- : [9b948720-c410-40e6-86ae-a600ada774da]   Rendered dashboard/index.html.erb within layouts/vueapp (Duration: 0.0ms | Allocations: 4)
I, [2022-09-27T13:11:11.640151 #1]  INFO -- : [9b948720-c410-40e6-86ae-a600ada774da]   Rendered layout layouts/vueapp.html.erb (Duration: 1.8ms | Allocations: 1566)
I, [2022-09-27T13:11:11.640474 #1]  INFO -- : [9b948720-c410-40e6-86ae-a600ada774da] Completed 200 OK in 9ms (Views: 2.3ms | ActiveRecord: 0.0ms | Allocations: 3842)
I, [2022-09-27T13:11:11.641428 #1]  INFO -- : source=rack-timeout id=9b948720-c410-40e6-86ae-a600ada774da timeout=15000ms service=15ms state=completed
I, [2022-09-27T13:11:12.580019 #1]  INFO -- : source=rack-timeout id=62c33de1-604b-42d9-83f7-a43f9c478916 timeout=15000ms state=ready
I, [2022-09-27T13:11:12.580752 #1]  INFO -- : [62c33de1-604b-42d9-83f7-a43f9c478916] Started GET "/auth/validate_token" for xxx.xxx.xxx.xxx at 2022-09-27 13:11:12 +0000
I, [2022-09-27T13:11:12.582454 #1]  INFO -- : [62c33de1-604b-42d9-83f7-a43f9c478916] Processing by DeviseOverrides::TokenValidationsController#validate_token as HTML
I, [2022-09-27T13:11:12.595353 #1]  INFO -- : [62c33de1-604b-42d9-83f7-a43f9c478916]   Rendered devise/token.json.jbuilder (Duration: 8.7ms | Allocations: 2398)
I, [2022-09-27T13:11:12.596824 #1]  INFO -- : [62c33de1-604b-42d9-83f7-a43f9c478916] Completed 200 OK in 14ms (Views: 8.3ms | ActiveRecord: 2.1ms | Allocations: 4059)
I, [2022-09-27T13:11:12.597516 #1]  INFO -- : source=rack-timeout id=62c33de1-604b-42d9-83f7-a43f9c478916 timeout=15000ms service=17ms state=completed
I, [2022-09-27T13:11:12.795532 #1]  INFO -- : source=rack-timeout id=454bc9c7-8f3c-47fe-8608-9bac5c133d3e timeout=15000ms state=ready
I, [2022-09-27T13:11:12.797003 #1]  INFO -- : [454bc9c7-8f3c-47fe-8608-9bac5c133d3e] Started GET "/api/v1/accounts/1/" for xxx.xxx.xxx.xxx at 2022-09-27 13:11:12 +0000
I, [2022-09-27T13:11:12.800245 #1]  INFO -- : [454bc9c7-8f3c-47fe-8608-9bac5c133d3e] Processing by Api::V1::AccountsController#show as JSON
I, [2022-09-27T13:11:12.800384 #1]  INFO -- : [454bc9c7-8f3c-47fe-8608-9bac5c133d3e]   Parameters: {"id"=>"1"}
I, [2022-09-27T13:11:12.827820 #1]  INFO -- : [454bc9c7-8f3c-47fe-8608-9bac5c133d3e]   Rendered api/v1/accounts/show.json.jbuilder (Duration: 18.8ms | Allocations: 4660)
I, [2022-09-27T13:11:12.828102 #1]  INFO -- : [454bc9c7-8f3c-47fe-8608-9bac5c133d3e] Completed 500 Internal Server Error in 28ms (ActiveRecord: 1.5ms | Allocations: 6881)
F, [2022-09-27T13:11:12.829718 #1] FATAL -- : [454bc9c7-8f3c-47fe-8608-9bac5c133d3e]
[454bc9c7-8f3c-47fe-8608-9bac5c133d3e] ActionView::Template::Error (undefined method `custom_attributes' for #<Account id: 1, name: "My Account", created_at: "2021-10-14 12:10:07.624289000 +0000", updated_at: "2021-11-09 10:35:26.927432000 +0000", locale: "de", domain: nil, support_email: "chat@mydomain.com", settings_flags: 0, feature_flags: 15, auto_resolve_duration: 30, limits: {}, status: nil>
Did you mean?  custom_attribute_definitions):
[454bc9c7-8f3c-47fe-8608-9bac5c133d3e]     1: json.auto_resolve_duration resource.auto_resolve_duration
[454bc9c7-8f3c-47fe-8608-9bac5c133d3e]     2: json.created_at resource.created_at
[454bc9c7-8f3c-47fe-8608-9bac5c133d3e]     3: if resource.custom_attributes.present?
[454bc9c7-8f3c-47fe-8608-9bac5c133d3e]     4:   json.custom_attributes do
[454bc9c7-8f3c-47fe-8608-9bac5c133d3e]     5:     json.plan_name resource.custom_attributes['plan_name']
[454bc9c7-8f3c-47fe-8608-9bac5c133d3e]     6:     json.subscribed_quantity resource.custom_attributes['subscribed_quantity']
[454bc9c7-8f3c-47fe-8608-9bac5c133d3e]
[454bc9c7-8f3c-47fe-8608-9bac5c133d3e] app/views/api/v1/models/_account.json.jbuilder:3
[454bc9c7-8f3c-47fe-8608-9bac5c133d3e] app/views/api/v1/accounts/show.json.jbuilder:1
[454bc9c7-8f3c-47fe-8608-9bac5c133d3e] app/controllers/api/v1/accounts_controller.rb:35:in `show'
[454bc9c7-8f3c-47fe-8608-9bac5c133d3e] app/controllers/concerns/request_exception_handler.rb:11:in `handle_with_exception'
[454bc9c7-8f3c-47fe-8608-9bac5c133d3e] app/controllers/concerns/switch_locale.rb:24:in `set_locale'
[454bc9c7-8f3c-47fe-8608-9bac5c133d3e] app/controllers/concerns/switch_locale.rb:11:in `switch_locale'
I, [2022-09-27T13:11:12.830241 #1]  INFO -- : source=rack-timeout id=454bc9c7-8f3c-47fe-8608-9bac5c133d3e timeout=15000ms service=35ms state=completed
```

**Environment**

Deployed into self hosted Kubernetes with helm as described in [the documentation](https://www.chatwoot.com/docs/self-hosted/deployment/helm-charts)

**Desktop (please complete the following information):**
 - OS: Windows 11
 - Browser Chrome
 - Version 105.0.5195.127 (Official Build) (64-bit)

**Additional context**

The same database works with Chatwoot  2.7.0 but not in 2.9.0

**Comment by pranavrajs** [comment]
@mxsxs2 Please run the database migrations after the update. https://www.chatwoot.com/docs/self-hosted/runbooks/migrate-chatwoot-database

**Comment by mxsxs2** [comment]
I followed those steps a couple of times just from the [Helm chart README](https://github.com/chatwoot/charts/tree/main/charts/chatwoot) and I always got the same results.

However, I got it running with after I slightly altered the upgrade process:

At step 5 I did not reset the `replicaCount` values and installed the new version with that. 
Then followed the steps up to 8.

Then I upgraded the config to reset the `replicaCount` values to default and run the final 9th step. Now it works as it should.

This worked as the migration pods were running BEFORE I restored the old DB back into postgres. With these modified steps now they run AFTER I restored the old DB back into postgres. 

@vishnu-narayanan Am I correct in doing this or am I missing something completely here?
